app/views/dependencies/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<% @title = params[:rubygem_id] %>
+<% @subtitle = @latest_version.try(:slug) %>
+<span id="deps_all"></span>
```

Comment:
What is this?

Reply:
Oh no. I had used that for debugging. I'll remove it.